biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -121,6 +121,11 @@ module FatesPlantHydraulicsMod
   ! 1 => use BC hydraulics; 
   ! 2 => use CX hydraulics
 
+  ! use a control to prevent model from quiting when encounter the water balance error
+  ! instead put it into an endless loop   (JD)
+  integer, public :: debug_JD = 1                        ! set to 0 to disable the 
```

Comment:
why endless loop?

Reply:
@xuchongang 
Yes, there is a variable in the code to control which model to use for plants. 
The endless loop is used to avoid the water balance errors that may cause model end in large ensemble I am running. As for the pull request, I will delete this section for the codes to be meagered.

Reply:
I can confirm that the soil P-V P-K curves still default to Campbell, and that the plants are controled by the parameter "hydr_htftype_node" in the parameter file.  The default parameter file still uses TFS P-V P-K for plants.
Resolving